### ISSUE TITLE
Were are the custom breakpoints?

### ISSUE THREAD

**Issue by chajadesign** [elementor/elementor#11461]
As Elementor being the most valued page builder in the Wordpress ecosystem it lacks a major functionality which is setting custom breakpoints. Maybe not for the free version but definitely for the pro version. The functionality of the three resolution options in Elementor gives you a mediocre toolset of working with different screen resolutions.

Custom breakpoints were already mentioned in the 2.1 beta release but did not make it still after all these new releases.
https://github.com/elementor/elementor/issues/4861

Your marketing statement on the website says:
**_Elementor's robust editor empowers professionals to edit and style WordPress websites visually, eliminating all the guesswork involved in writing code._**

How are you going to empower professionels if you are not implementing such important functionality? I have watched several youtube video's of your channel and read about your experts platform. But how do these professional courses compare to this missing functionality which will deeply enhance the professionalization of developing a website with Elementor?

So my question is if development of Elementor is going to bring these needed developer enhancements.

**Comment by shilo-ey** [comment]
Hi @chajadesign,

Thanks for taking the time to share your inputs.

We are aware of this request and actively working on allowing this feature, which had some technical restraints. 

It should be released in the few upcoming major versions.

**Comment by shilo-ey** [comment]
Duplicate of #2043

**Comment by chajadesign** [comment]
Hi @shilo-ey,

Thank you for replying so swiftly.
That sounds promising. I am looking forward to this implementation.

**Comment by chajadesign** [comment]
Hi @shilo-ey,

Do you mean that it will be implemented in version 3 this year?

**Comment by shoutcloudstudios** [comment]
Yes, @shilo-ey, can you provide some clue in terms of timing? We won't hold you to an exact date. We just want to know if you mean that it could possibly happen this year. Thanks!

**Comment by mdca** [comment]
> Hi @chajadesign,
> 
> Thanks for taking the time to share your inputs.
> 
> We are aware of this request and actively working on allowing this feature, which had some technical restraints.
> 
> It should be released in the few upcoming major versions.

Major version as in 4.x, 5.x or 11.x?

**Comment by chajadesign** [comment]
If not released in version 3 it will at least take one and a half year before we can see this happen.
Or maybe longer. So we probably will have to look forward to an elementor addon which can accomplish this earlier.

**Comment by mdca** [comment]
> If not released in version 3 it will at least take one and a half year before we can see this happen.
> Or maybe longer. So we probably will have to look forward to an elementor addon which can accomplish this earlier.

Or just move to another builder and do not extend Expert License. That's what I'll do.

**Comment by chajadesign** [comment]
@mdca Do you already have an alternative for elementor which does have this feature?

**Comment by mdca** [comment]
@chajadesign unfortunately not, but Oxygen builder has more breakpoints and they are adjustable. It will be a bit complicated to move new projects to Oxygen, but I'm done with Elementor.

**Comment by chajadesign** [comment]
The same story is for Beaver Builder which was the innovator for the pagebuilder concept. But now it is far behind elementor and nothing new is happening there. But still they keep prices high although elementor has more functionalities for less money.
They even acknowledged that elementor has profited from their concept. So you say the competition is 'better' and let your own product die slowly.
You get the feeling that business models are drying up and they only want to sustain support and basic updates.
I will check oxygen ... thanks

**Comment by mdca** [comment]
> The same story is for Beaver Builder which was the innovator for the pagebuilder concept. But now it is far behind elementor and nothing new is happening there. But still they keep prices high although elementor has more functionalities for less money.
> They even acknowledged that elementor has profited from their concept. So you say the competition is 'better' and let your own product die slowly.
> You get the feeling that business models are drying up and they only want to sustain support and basic updates.
> I will check oxygen ... thanks

Or you get visual pleasing features already covered in large by 3rd party plugins. Its really sad.
